components/sections/homepage/Services/index.jsx

```diff
@@ -6,9 +6,9 @@ import ServiceItem from "./ServiceItem";
 const Services = () => {
   return (
     <Wrapper>
-      <div className="px-6.5 md:px-0 mt-17.5 md:mt-15 xl:mt-16">
+      <div className="mt-2.75 md:mt-15 xl:mt-16">
         <Heading>What we do</Heading>
-        <div className="pt-16.25 md:pt-20.5 xl:pt-31.25 flex flex-col gap-32">
+        <div className="pt-9 md:pt-20.5 xl:pt-26.75 flex flex-col gap-16 md:gap-32 overflow-hidden">
```

Comment:
We've done like so because in the feature when those services data will be coming from an API, certainly they won't have those properties (`reverse` and `fadeDir`) included. This way prevents us from changing again that logic in the feature in case it happens to fetch those data from an API.

Reply:
I think your approach here makes sense if you know there is a possibility that those services data may be coming from an API in the future

From your services mocked data you are currently using it looks like there are `width` and `height` properties for each service. Since you want an approach that will not require you to change a lot in your current logic. Are you sure those properties will be included in the data that will be coming from the API?

Reply:
Yeah, we got an advice of putting those properties there (`width` & `height`)  from Yannick. And for those two properties, it's possible they can come from the API along with the image data